server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/FlowTaskCallBackApprovalService.java

```diff
@@ -103,7 +103,7 @@ private void completeTask(long flowInstanceId, long flowTaskInstanceId, FlowNode
         FlowInstanceEntity flowInstance = getFlowInstance(flowInstanceId);
         FlowableElement flowableElement = getFlowableElementOfUserTask(flowTaskInstanceId);
 
-        Await.await().timeout(60).timeUnit(TimeUnit.SECONDS)
+        Await.await().timeout(60).timeUnit(TimeUnit.SECONDS).period(1).periodTimeUnit(TimeUnit.SECONDS)
```

Comment:
Could you please explain why we need this line of code?

Reply:
This code is for task framework by @krihy . It would block 5s to wait approval node to be created. I changed this to wait only 1s .
But this modification is only for optimizing the code. The fix that truly works is another in  `BaseODCFlowTaskDelegate` .

Reply:
Yes I know that your fix is in `BaseODCFlowTaskDelegate`, so I wonder what's the effect of this line of code.
Is this change necessary? Or will it affect the task framework? @krihy